packages/react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -116,16 +119,33 @@ export default class Tooltip extends AutoControlledComponent<TooltipProps, Toolt
     position: 'above',
     mouseLeaveDelay: 500,
     pointing: true,
+    accessibility: tooltipBehavior,
   }
 
   static autoControlledProps = ['open']
 
   pointerTargetRef = React.createRef<HTMLElement>()
   triggerRef = React.createRef<HTMLElement>()
   contentRef = React.createRef<HTMLElement>()
-
   closeTimeoutId
 
+  actionHandlers = {
+    close: e => {
+      this.setTooltipOpen(false, e)
+      e.stopPropagation()
+      e.preventDefault()
+    },
+  }
+
+  static getAutoControlledStateFromProps(
+    props: TooltipProps,
+    state: TooltipState,
+  ): Partial<TooltipState> {
+    return {
+      tooltipId: getOrGenerateIdFromShorthand('tooltip-content-', props.content, state.tooltipId),
```

Comment:
```suggestion
      tooltipId: getOrGenerateIdFromShorthand('tooltip-content-', props.content, state.contentId),
```

It looks like a `contentId`...